// @eox/ui/lib/vuetify/blueprint.js
export const eox = {
  defaults: {
    VAppBar: {
      flat: true,
      VBreadcrumbs: {
        VBreadcrumbsItem: {
          class: "text-white",
        },
        VBreadcrumbsDivider: {
          class: "text-white",
        },
      },
    },
    VAutocomplete: {
      variant: "filled",
    },
    VBanner: {
      color: "primary",
    },
    VBottomSheet: {
      contentClass: "rounded-t-xl overflow-hidden",
    },
    VBtn: {
      color: "primary",
      rounded: "xl",
      class: "text-none",
    },
    VBtnGroup: {
      rounded: "xl",
      VBtn: { rounded: null },
    },
    VCard: {
      rounded: "lg",
    },
    VCheckbox: {
      color: "secondary",
      inset: true,
    },
    VChip: {
      rounded: "sm",
    },
    VCombobox: {
      variant: "filled",
    },
    VNavigationDrawer: {
      // VList: {
      //   nav: true,
      //   VListItem: {
      //     rounded: 'xl',
      //   },
      // },
    },
    VSelect: {
      variant: "filled",
    },
    VSlider: {
      color: "primary",
    },
    VTabs: {
      color: "primary",
    },
    VTextarea: {
      variant: "filled",
    },
    VTextField: {
      variant: "filled",
    },
    VToolbar: {
      VBtn: {
        color: null,
      },
    },
  },
};
